contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-epilogue.y.c

```diff
@@ -233,35 +233,39 @@ TsqlFunctionConvert(TypeName *typename, Node *arg, Node *style, bool try, int lo
 Node *
 TsqlFunctionIdentityInto(TypeName *typename, Node *seed, Node *increment, int location)
 {
-	Node *result;
-	List *args;
-	int32 typmod;
-	Oid type_oid;
-	Oid base_oid;
-	typenameTypeIdAndMod(NULL, typename, &type_oid, &typmod);
-	base_oid = getBaseType(type_oid);
-	switch (base_oid)
-	{
-		case INT2OID:
-			args = list_make3((Node *)makeIntConst((int)type_oid, location), seed, increment);
-			result = (Node *)makeFuncCall(TsqlSystemFuncName("identity_into_smallint"), args, COERCE_EXPLICIT_CALL, location);
-			break;
-		case INT4OID:
-			args = list_make3((Node *)makeIntConst((int)type_oid, location), seed, increment);
-			result = (Node *)makeFuncCall(TsqlSystemFuncName("identity_into_int"), args, COERCE_EXPLICIT_CALL, location);
-			break;
-		case INT8OID:
-		case NUMERICOID:
-			args = list_make3((Node *)makeIntConst((int)INT8OID, location), seed, increment); /* Used bigint internally for decimal and numeric as well*/
-			result = (Node *)makeFuncCall(TsqlSystemFuncName("identity_into_bigint"), args, COERCE_EXPLICIT_CALL, location);
-			break;
-		default:
-			ereport(ERROR,
-					(errcode(ERRCODE_SYNTAX_ERROR),
-					errmsg("Parameter or variable '' has an invalid data type.")));
-			break;
-	}
-	return result;
+	ereport(ERROR, (errcode(ERRCODE_SYNTAX_ERROR), errmsg("syntax error near 'identity'")));
+
+	// Node *result;
+	// List *args;
```

Comment:
Two small nits: IMO an error message like "Insert Into with Identity is not yet supported" might make it more clear that this is a valid syntax, just one that's not yet supported in BBF. 
And, since we can still have this code saved in a separate branch, I'd suggest just deleting this instead of commenting it out.

Reply:
Done both
DBE suggested message  "IDENTITY() function in SELECT INTO is not supported."